uwblueprint/supportive-housing
# added update service

## Notion task link
[[](https://www.notion.so/uwblueprintexecs/Update-Tags-Route-331aea6e13af4e4aa6c50a3a1a062848?pvs=4)]

[Update Tags Route](https://www.notion.so/uwblueprintexecs/2792c3504d744d2b90ff967c32d00961?v=d33629815ecf477ab4e1ff4e33774c32)



## Implementation description
* Implemented the route that updates a given tag from the database



## Steps to test
1. Insert sample tags into the database. Start by booting up the database using the `bash ./scripts/exec-db.sh command`. Then, insert a few tags into your database. Example insertion: `insert into tags (tag_id, name, status) values (3, 'test3', 'Active');`. Note that the status must be either "Active" or "Deleted"
2. Then, try updating the name of that tag (since only the name should be updated) using postman. There is a sample body in the REST APIs in Notion
3. Also, try updating a tag to have the same name of another tag. This should fail as each name should be unique


## What should reviewers focus on?
* Focus on functionality and making sure that no two tags can be updated to have the same name


## Checklist
